src/components/elements/EditableDiv.js

```diff
@@ -0,0 +1,30 @@
+import "bootstrap-icons/font/bootstrap-icons.css";
+import classes from "./EditableDiv.module.css";
+
+function EditableDiv(props) {
```

Comment:
I wouldn't tie the name of a component to implementation detail (div), maybe use something more generic?

Reply:
Changed to 'Box' instead